README.md

```diff
@@ -32,13 +32,9 @@ Update user with ID 4? Send a `POST` request to `/wp-json/wp/v2/users/4`. Get al
 posts with the search term "awesome"? `GET /wp-json/wp/v2/posts?filter[s]=awesome`.
 It's that easy.
 
-WP API exposes a simple yet easy interface to WP Query, the posts API, post
-meta API, users API, revisions API and many more. Chances are, if you can do
-it with WordPress, WP API will let you do it.
+The WordPress REST API exposes a simple yet easy interface to WP Query, the posts API, post meta API, users API, revisions API and many more. Chances are, if you can do it with WordPress, API will let you do it.
```

Comment:
"simple yet powerful" sounds good. Alternatively we could just say "simple" or "easy to use"...but I feel like both of those are questionably accurate by themselves.